Methods.h

```diff
@@ -0,0 +1,50 @@
+#include "Classes.h"
+using namespace std;
+
+void Smartphone::add(float p, string n, float M, float A, float B)
+{
+    name = n;
+    price = p;
```

Comment:
Pending

You are using an unclear shift name.

You should:
smartphone_model_name_ = model_name;
smartphone_price_ = price;